api/nodes/serializers.py

```diff
@@ -164,6 +164,11 @@ class NodeSerializer(JSONAPISerializer):
         related_view_kwargs={'node_id': '<forked_from_id>'}
     )
 
+    templated_from = RelationshipField(
+        related_view=lambda n: 'registrations:registration-detail' if getattr(n, 'template_node', None) else 'nodes:node-detail',
```

Comment:
I'm reasonably certain the getattr there should be grabbing 'is_registration' to determine if it's a registration or a node.